### DISCUSSION TITLE
What is the meaning of the keyword `auto`?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#3015] [General]
I encountered it on the section `Размерные и ссылочные типы` of the help file. I have read the whole section, but I have to admit that I'm still not get it. The writing is difficult to understand to me.

Update: It's already covered in `Автоклассы` of `Классы`. Sorry.

**Comment by SunSerega** [comment]
There are also `auto`-properties. In the help file there is "Index" tab, where you can search for `auto`.

**Reply by ghost** [reply]
Not work in the web version: https://pascalabc.net/downloads/pabcnethelp/index.htm

I have to use the web version to have my machine translator to work. It's a Chrome extension.

**Reply by SunSerega** [reply]
Ah, I see, it seems pretty outdated in many places. You can open help in IDE using F1. You can also place the cursor on a word you want to search for and press F1 - it will open the right help page (can even open the MSDN page for .Net stuff, tho built-in browser is still the IE).
